docker/complete/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+EXPECTED_SCRIPT_DIR=docker/complete
+EXPECTED_BINARY_DIR=bin
+
+set -ex
+
+script_dir=$(cd $(dirname $0) && pwd -P)
+
+# defence against lazy refactoring
+[[ "$script_dir" != *"$EXPECTED_SCRIPT_DIR" ]] && echo "Build script has moved! Expected location: $EXPECTED_SCRIPT_DIR" && exit 1
+
+project_dir=$(cd "${script_dir}/../.." && pwd -P)
+cd ${project_dir}
+
+# create temp dir in project dir to avoid permission issues
+WORKSPACE=$(env TMPDIR=$PWD mktemp -d -t "k8sm-workspace")
+echo "Workspace created: $WORKSPACE"
+
+cleanup() {
+    rm -rf ${WORKSPACE}
+    echo "Workspace deleted: $WORKSPACE"
+}
+trap 'cleanup' EXIT
+
+mkdir ${WORKSPACE}/bin
+
+echo "Building kubernetes-mesos binaries"
+docker run --rm -v ${WORKSPACE}/bin:/target mesosphere/kubernetes-mesos:build
```

Comment:
this will always do a fresh `git clone`. you should be able to avoid that (if you want) by adding `-v /path/to/already-checked-out-repo:/snapshot` to the docker run arg list.